contracts/strategies/traderjoe/TraderJoe.sol

```diff
@@ -31,47 +30,135 @@ contract TraderJoe is UUPSUpgradeable, StrategyOwnablePausableBaseUpgradeable {
 
     function initialize(
         StrategyArgs calldata strategyArgs,
-        ITraderJoeRouter router,
-        ITraderJoeMasterChef masterChef,
-        ITraderJoePair lpToken,
-        IERC20Upgradeable joeToken
+        ITraderJoeLBPair lbPair,
+        ITraderJoeLBRouter lbRouter,
+        uint256 binStep,
+        uint256[] calldata binIds,
+        uint256[] calldata binAllocationsX,
+        uint256[] calldata binAllocationsY
     ) external initializer {
+        checkBinIds(binIds);
+        checkBinAllocations(binAllocationsX);
+        checkBinAllocations(binAllocationsY);
```

Comment:
if we do any checking here, can we be throrough and check if bindIds.length==binAllocationsX.length==binAllocationsY.length

Reply:
I didn't add it because we can use the same config in the test and production and if we pass different length of them them CI will fail. But okay, I'll add it.